libraries/AP_Logger/AP_Logger.cpp

```diff
@@ -176,7 +176,16 @@ const AP_Param::GroupInfo AP_Logger::var_info[] = {
     // @Increment: 0.1
     // @User: Standard
     AP_GROUPINFO("_DARM_RATEMAX",  11, AP_Logger, _params.disarm_ratemax, 0),
-    
+
+    // @Param: _MAX_FILES
+    // @DisplayName: Maximum number of log files
+    // @Description: This sets the maximum number of log file that will be written on dataflash or sd card before starting to rotate log number.
+    // @Range: 1 32767
```

Comment:
will 1 work?

Reply:
nice one. 
Probably need to make a test suite for this then